front/src/applications/stdcmV2/views/StdcmViewV2.tsx

```diff
@@ -3,16 +3,28 @@ import React, { useState } from 'react';
 import { useSelector } from 'react-redux';
 
 import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import { STDCM_REQUEST_STATUS } from 'applications/stdcm/consts';
+import useStdcm from 'applications/stdcm/hooks/useStdcm';
+import type { StdcmV2SuccessResponse } from 'applications/stdcm/types';
 import { useOsrdConfSelectors } from 'common/osrdContext';
+import { getTrainScheduleV2Activated } from 'reducers/user/userSelectors';
 
 import StdcmConfig from '../components/StdcmConfig';
 import StdcmHeader from '../components/StdcmHeader';
+import StdcmResults from '../components/StdcmResults';
 import type { StdcmSimulationResult } from '../types';
 
 const StdcmViewV2 = () => {
+  const { launchStdcmRequest, cancelStdcmRequest, currentStdcmRequestStatus, stdcmV2Results } =
+    useStdcm();
+
+  const isPending = currentStdcmRequestStatus === STDCM_REQUEST_STATUS.pending;
```

Comment:
Move the computation of isPending in useStdcm directly, because we will eventually remove currentStdcmRequestStatus

Reply:
fix